CkanConnector/package.json

```diff
@@ -4,9 +4,11 @@
   "description": "MAGDA CKAN Connector",
   "main": "index.js",
   "scripts": {
-    "build": "tsc",
+    "build": "export REGISTRY_URL=http://localhost:6100/;npm run generateSwaggerApis && tsc",
```

Comment:
Oh also we might not want to `npm run generateSwaggerApis` because it requires the registry already be running.  In master the generated code is checked in now, avoiding the problem you ran into.